unit_test/test_spdm_requester/get_measurement_extension_log.c

```diff
@@ -0,0 +1,789 @@
+                                              false, spdm_response_size,
+                                              spdm_response, response_size,
+                                              response);
+
+        calling_index++;
+        if (calling_index == count) {
+            calling_index = 0;
+            spdm_mel_len = 0;
+        }
+    }
+        return LIBSPDM_STATUS_SUCCESS;
+
+    case 0x3:
+    case 0x4:
+    case 0x5:
+    {
+        spdm_measurement_extension_log_response_t *spdm_response;
+        size_t spdm_response_size;
+        size_t transport_header_size;
+        uint16_t portion_length;
+        uint16_t remainder_length;
+        spdm_measurement_extension_log_dmtf_t *spdm_mel;
+
+        static size_t count = 0;
+        static size_t calling_index = 0;
+
+        if (calling_index == 0) {
+            count = m_mel_buffer_size / LIBSPDM_MAX_MEL_BLOCK_LEN + 1;
+        }
+
+        spdm_mel = (spdm_measurement_extension_log_dmtf_t *)m_mel_buffer;
+
+        transport_header_size = LIBSPDM_TEST_TRANSPORT_HEADER_SIZE;
+        spdm_response = (void *)((uint8_t *)*response + transport_header_size);
+
+        spdm_response->header.spdm_version = SPDM_MESSAGE_VERSION_13;
+        spdm_response->header.request_response_code = SPDM_MEASUREMENT_EXTENSION_LOG;
+        spdm_response->header.param1 = 0;
+        spdm_response->header.param2 = 0;
+
+        if(m_mel_buffer_size > LIBSPDM_MAX_MEL_BLOCK_LEN) {
+            portion_length = LIBSPDM_MAX_MEL_BLOCK_LEN;
+            remainder_length =
+                (uint16_t)(m_mel_buffer_size - LIBSPDM_MAX_MEL_BLOCK_LEN);
+        } else {
+            portion_length  = m_mel_buffer_size;
+            remainder_length = 0;
+        }
+
+        spdm_response->portion_length = portion_length;
+        spdm_response->remainder_length = remainder_length;
+
+        libspdm_copy_mem(spdm_response + 1,
+                         (size_t)(*response) + *response_size - (size_t)(spdm_response + 1),
+                         (uint8_t *)spdm_mel +
+                         LIBSPDM_MAX_MEL_BLOCK_LEN * calling_index,
+                         portion_length);
+
+        spdm_response_size = sizeof(spdm_measurement_extension_log_response_t) + portion_length;
+
+        libspdm_transport_test_encode_message(spdm_context, NULL, false,
+                                              false, spdm_response_size,
+                                              spdm_response, response_size,
+                                              response);
+
+        m_mel_buffer_size -= portion_length;;
+        calling_index++;
+
+        if (calling_index == count) {
+            calling_index = 0;
+            m_mel_buffer_size = 0;
+        }
+    }
+        return LIBSPDM_STATUS_SUCCESS;
+
+    default:
+        return LIBSPDM_STATUS_RECEIVE_FAIL;
+    }
+}
+
+/**
+ * Test 1: message could not be sent
+ * Expected Behavior: get a LIBSPDM_STATUS_SEND_FAIL, with no CERTIFICATE messages received
+ **/
```

Comment:
`with no CERTIFICATE messages received`
Here the comment is not right, it should be `measurement_extension_log`, not `CERTIFICATE`.
Wenxing's latest PR already corrected this typo.